.env

```diff
@@ -0,0 +1,12 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=root
+MYSQLDB_DATABASE=book_store
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
+
+JWT_EXPIRATION=600000
+JWT_SECRET=1q2w3e4r5t6y7u8i9o0pqawsedrftgyhujikolpazsxdcfvgbhnjmkl
```

Comment:
Add this file to `.gitignore`. Putting a file with credentials in a public repository is not secure. Instead, add a file `.env.sample` with a list of variables without secure values.